AK/Utf32View.h

```diff
@@ -52,7 +52,7 @@ class Utf32CodePointIterator {
 private:
     Utf32CodePointIterator(u32 const* ptr, size_t length)
         : m_ptr(ptr)
-        , m_length((ssize_t)length)
+        , m_length(static_cast<ssize_t>(length))
```

Comment:
Why is the length signed anyway?

Reply:
:thinking:

Reply:
It's a marker value. Let's not :yaktangle: this PR even more.